docs/user-guides/observability-ui-plugins.md

```diff
@@ -52,3 +80,52 @@ metadata:
 spec:
   type: DistributedTracing
 ```
+
+#### Feature Matrix
+
+| __COO Version__ |   __OCP Versions__  | __Features__                                          |
+| --------------- | ------------------- | ----------------------------------------------------- |
+| 0.3.0+          | 4.11+               | _No features configuration, just core functionality_  |
+
+### Logging
+
+The plugin adds various Logging UI functionalities to the OpenShift web console. The core functionality of this plugin adds a new admin perspective tab `Observe > Logging`. This page includes query and log filters, the list of logs, and a histogram showing log frequency by severity. The logs can be filtered by an number of tags, such as `tenant` and `namespace`. The page also has controls for the length of time to query over, the refresh rate of the logging page, and whether to show kubernetes resource information in the results, such as `pod` and `container`. The reults section shows a list of collapsed logs, which can then be expanded to show more detailed information for each log.
+
+When a __TroubleshootingPanel__ `UIPlugin` is deployed the plugin will connect the [Korrel8r](https://github.com/korrel8r/korrel8r) service and add direct links from the admin perspective `Observe > Logging` page to the `Observe > Metrics` page with a correlated PromQL query. It will also add a "See Related Logs" link from the admin perspective `Observe > Alerting` on an alerting detail page to the `Observe > Logging` page with a correlated filter set selected.
```

Comment:
```suggestion
When a __TroubleshootingPanel__ `UIPlugin` is deployed the plugin will connect the [Korrel8r](https://github.com/korrel8r/korrel8r) service and add direct links from the admin perspective `Observe > Logs` page to the `Observe > Metrics` page with a correlated PromQL query. It will also add a "See Related Logs" link from the admin perspective `Observe > Alerting` on an alerting detail page to the `Observe > Logs` page with a correlated filter set selected.
```